harness/determined/cli/user.py

```diff
@@ -306,45 +292,41 @@ def revoke_token(args: argparse.Namespace) -> None:
     sess = cli.setup_session(args)
-    current_user, token_user = sess.username, args.username
-    request, handler = None, None
-    if token_user is None or token_user == current_user:
-        user_id = bindings.get_GetUserByUsername(session=sess, username=current_user).user.id
-    else:
-        user_id = bindings.get_GetUserByUsername(session=sess, username=token_user).user.id
+    username = args.username or sess.username
+    user_id = bindings.get_GetUserByUsername(session=sess, username=username).user.id
+
+    request = None
     request = bindings.v1PostAccessTokenRequest(
-        lifespan=args.lifespan, userId=user_id, description=args.description
+        lifespan=args.expiration_duration, userId=user_id, description=args.description
     )
-    handler = functools.partial(bindings.post_PostAccessToken, userId=user_id)
-    resp = handler(session=sess, body=request).to_json()
+    resp = bindings.post_PostAccessToken(sess, userId=user_id, body=request).to_json()
 
-    outputString = None
+    output_string = None
     if args.yaml:
-        outputString = util.yaml_safe_dump(resp, default_flow_style=False)
+        output_string = util.yaml_safe_dump(resp, default_flow_style=False)
     elif args.json:
-        outputString = json.dumps(resp, indent=2)
+        output_string = json.dumps(resp, indent=2)
     else:
-        outputString = resp["token"]
+        output_string = f'{resp["token"]}\n{resp["tokenId"]}'
 
-    if args.file_path:
-        with open(args.file_path, "w") as file:
-            file.write(outputString)
+    if args.output_file:
+        with open(args.output_file, "w") as file:
+            file.write(output_string)
     else:
-        print(outputString)
+        print(output_string)
 
 
-def update_token(args: argparse.Namespace) -> None:
-    sess = cli.setup_session(args)
-    try:
-        if args.description and args.token_id:
-            request = bindings.v1PatchAccessTokenRequest(
-                tokenId=args.token_id, description=args.description, setRevoked=False
-            )
-            resp = bindings.patch_PatchAccessToken(sess, body=request, tokenId=args.token_id)
-        print(json.dumps(resp.to_json(), indent=2))
-    except api.errors.NotFoundException:
-        raise errors.CliError("Token not found")
-    print("Successfully updated token with ID: {}".format(args.token_id))
+def login_with_token(args: argparse.Namespace) -> None:
+    unauth_session = api.UnauthSession(master=args.master, cert=cli.cert)
+    auth_headers = {"Authorization": f"Bearer {args.token}"}
+    user_data = unauth_session.get("/api/v1/me", headers=auth_headers).json()
+    if 'user' in user_data and 'username' in user_data['user']:
+        username = user_data.get('user').get('username')
+
+    token_store = authentication.TokenStore(args.master)
+    token_store.set_token(username, args.token)
+    token_store.set_active(username)
+    print("Authenticated as {}.".format(username))
```

Comment:
nit (style): Would it be worth using an f-string literal here like:
```suggestion
    print(f"Authenticated as {username}.")
```